src/node-capnp/capnp.cc

```diff
@@ -1280,23 +1280,39 @@ struct FromJsConverter {
         break;
       }
       case capnp::schema::Type::ANY_POINTER:
-        KJ_IF_MAYBE(reader, unwrapReader(js)) {
-          return orphanage.newOrphanCopy(*reader);
-        } else KJ_IF_MAYBE(buffer, unwrapBuffer(js)) {
-          kj::Array<capnp::word> scratch;
-          kj::ArrayPtr<const capnp::word> words;
-          if (reinterpret_cast<uintptr_t>(buffer->begin()) % sizeof(capnp::word) != 0) {
-            // Array is not aligned.  We have to make a copy.  :(
-            scratch = kj::heapArray<capnp::word>(buffer->size() / sizeof(capnp::word));
-            memcpy(scratch.begin(), buffer->begin(), buffer->size());
-            words = scratch;
-          } else {
-            // Yay, array is aligned.
-            words = kj::arrayPtr(reinterpret_cast<const capnp::word*>(buffer->begin()),
-                                 buffer->size() / sizeof(capnp::word));
+        if (type.whichAnyPointerKind() == capnp::schema::Type::AnyPointer::Unconstrained::CAPABILITY) {
+          if (js->IsNull()) {
+            auto cap = capnp::Capability::Client(nullptr);
+            capnp::DynamicCapability::Client dynamicCap(kj::mv(cap));
+            return orphanage.newOrphanCopy(dynamicCap);
+          } else KJ_IF_MAYBE(cap, Wrapper::tryUnwrap<capnp::DynamicCapability::Client>(js)) {
+            return orphanage.newOrphanCopy(*cap);
+          } else if (!localCapType.IsEmpty()) {
+            // TODO(someday): handle schemaless local caps
```

Comment:
I'm not sure how to make an InterfaceSchema from a nullptr, when i try to do `capnp::InterfaceSchema schema(nullptr)`, I get:
```
  /ekam-provider/canonical/node-capnp/capnp.cc:1294:38: error: no matching constructor for initialization of 'capnp::InterfaceSchema'
                capnp::InterfaceSchema schema(nullptr);
                                       ^      ~~~~~~~
  /ekam-provider/c++header/capnp/schema.h:438:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'nullptr_t' to 'const capnp::
    InterfaceSchema' for 1st argument
  class InterfaceSchema: public Schema {
        ^
  /ekam-provider/c++header/capnp/schema.h:438:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion from 'nullptr_t' to 'capnp::
    InterfaceSchema' for 1st argument
  class InterfaceSchema: public Schema {
        ^
  /ekam-provider/c++header/capnp/schema.h:465:3: note: candidate constructor not viable: no known conversion from 'nullptr_t' to 'capnp::Schema' for 1st argument
    InterfaceSchema(Schema base): Schema(base) {}
```